hadoop-hdds/docs/content/feature/Quota.md

```diff
@@ -32,11 +32,16 @@ So far, we know that Ozone allows users to create volumes, buckets, and keys. A
 1. Storage Space level quota
 
 Administrators should be able to define how much storage space a Volume or Bucket can use. The following Settings for Storage space quota are currently supported：
+
 a. By default, the quota for volume and bucket is not enabled.
+
 b. When volume quota is enabled, the total size of bucket quota cannot exceed volume.
```

Comment:
Minor:
When volume quota is enabled, the total quota of the buckets, cannot exceed the volume quota.